### PR TITLE
PT-19: Add items per page drop down

### PR DESCRIPTION
**Pull request by mitko100**

### CODE REVIEW

**Comment by mitko100** [comment]
@ValBaliyska, please review.

**Comment by ValBaliyska** [comment]
> @ValBaliyska, please review.

I will review it today.

**Comment by ValBaliyska** [comment]
I really like this new functionality. It would be great if we can have Last Week and Current Week also. Btw Reports look broken. And one more thing, Custom is not yet implemented, right?

**Comment by mitko100** [comment]
> I really like this new functionality. It would be great if we can have Last Week and Current Week also. Btw Reports look broken. And one more thing, Custom is not yet implemented, right?

Yes, the report is broken. 
Did you switch to the new branch before running it? 
**Note**: The branch is **AddItemsPerPageDropDown**

**Comment by mitko100** [comment]
The new-look is: 
image

**Comment by ValBaliyska** [comment]
> > I really like this new functionality. It would be great if we can have Last Week and Current Week also. Btw Reports look broken. And one more thing, Custom is not yet implemented, right?
> 
> Yes, the report is broken. I will add Last Week and Current Week. 👍
> Did you switch to the new branch before running it?
> **Note**: The branch is **AddItemsPerPageDropDown**

Sorry, I was on the wrong branch. Everythink looks good, I really like the look and feel now. Moving Add button at the top left corner is great idea. Will continue testing for eventually popped bugs.

**Comment by mitko100** [comment]
The plan for the button is to make it more round close to circle button. But this will be on the next stage.
Review the code, and if there are no comments - approve it. Thank you!

File: PennyTracker.BlazorServer/ViewModels/ExpensesTableViewModel.cs [hunk 1]
```diff
@@ -43,13 +50,27 @@ public class ExpensesTableViewModel : IExpensesTableViewModel
             this.notificationService = notificationService;
             this.dialogService = dialogService;
 
-            this.Periods = new List<string> { "Last Month", "Current Month", "Custom" };
-            this.SelectedPeriod = this.Periods.Skip(1).First();
+            this.eventAggregator.GetEvent<AddTransactionEvent>()
+                .Subscribe(async () => await this.OnButtonAddClickAsync());
+
+            this.ItemsPerPage = new List<int> { 5, 8, 10, 50, 100 };
```
Comment (ValBaliyska) [inline PennyTracker.BlazorServer/ViewModels/ExpensesTableViewModel.cs]: Don't like this 8 here, lets have just 0s, 1s and 5s. For example: 5, 10, 15, 50, 100.

Comment (mitko100) [reply PennyTracker.BlazorServer/ViewModels/ExpensesTableViewModel.cs]: Yes, will remove it. 
I put it to facilitate testing :)

Comment (ValBaliyska) [reply PennyTracker.BlazorServer/ViewModels/ExpensesTableViewModel.cs]: ok, let's stay then 10, 25, 50, 100.

**Pull request merged**